### PR TITLE
Release 12.0.3

### PR DESCRIPTION
**Pull request by r0mant**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+### Security fixes
+
+* Fixed issue with malicious SQL Server packet being able to cause proxy crash. [#21638](https://github.com/gravitational/teleport/pull/21638)
+* Fixed issue with session terminated after a short delay instead of being immediately paused when moderator leaves. [#21974](https://github.com/gravitational/teleport/pull/21974)
+
+### Other improvements and bug fixes
+
+* Fixed issue with not being able to see any pods with an active access request. [#22196](https://github.com/gravitational/teleport/pull/22196)
+* Fixed issue with remote cluster state not always being correctly updated. [#22088](https://github.com/gravitational/teleport/pull/22088)
+* Fixed heartbeat errors from database servie. [#22087](https://github.com/gravitational/teleport/pull/22087)
```
Comment (zmb3) [inline CHANGELOG.md:16]:
```suggestion
* Fixed heartbeat errors from database service. [#22087](https://github.com/gravitational/teleport/pull/22087)
```

**Review by zmb3 — APPROVED** [review]

**Review by russjones — APPROVED** [review]

**Review by mdwn — APPROVED** [review]

**Comment by SMillerDev** [comment]
Any reason this isn't marked as a GitHub release?

**Comment by r0mant** [comment]
@SMillerDev We had an issue with 12.0.3 publishing and discovered a regression in it as well. We'll publish 12.0.4 today instead with the fix.

**Comment by chenrui333** [comment]
@r0mant thanks for the new release, but it looks like it did not make to the `latest` label either, can you take a look? Thanks!

**Comment by chenrui333** [comment]
Thanks @r0mant!! Appreciate it.

**Pull request merged**